src/DocumentFormat.OpenXml.Framework/Packaging/OpenXmlPackage.FlatOpc.cs

```diff
@@ -127,7 +130,7 @@ private static XElement GetBinaryPartContentsAsXml(PackagePart part)
                 new XElement(Pkg + "binaryData", ToChunkedBase64String(part)));
```

Comment:
Would it be better to use `String.Concat()` here instead?

Reply:
Probably - there are many issues with the FlatOpc implementations regarding performance... this PR was focused on replacing things with the ipackage rather than fixing anything else.